automatic/freecad/tools/chocolateyInstall.ps1

```diff
@@ -1,17 +1,62 @@
-﻿$ErrorActionPreference = 'Stop';
+$ErrorActionPreference = 'Stop';
+
+$toolsDir = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+
+# Checking for Package Parameters
+$pp = Get-PackageParameters
+if (!$pp['UnzipLocation']) { $pp['UnzipLocation'] = "$toolsDir" }
```

Comment:
Please use dot notation as there is no need for extra markup. I.E `$pp.UnzipLocation` rather then `$pp['UnzipLocation']`.